frontend/pages/hosts/details/cards/AgentOptions/AgentOptions.tsx

```diff
@@ -19,7 +22,9 @@ const AgentOptions = ({
   wrapFleetHelper,
   isChromeOS = false,
 }: IAgentOptionsProps): JSX.Element => {
-  const classNames = classnames(baseClass, "section", "osquery");
+  const classNames = classnames(baseClass, {
+    [`${baseClass}--chrome-os`]: isChromeOS,
```

Comment:
What's our pattern for "--" versus "__"? My understanding was "--" is for updating states during interactivity (error, active), while "__" is for data-specified differenced, like this. Wdyt?

Reply:
Sgtm! I'll update later day